posthog/models/feature_flag.py

```diff
@@ -62,6 +62,7 @@ def get_analytics_metadata(self) -> Dict:
             "has_rollout_percentage": any(condition.get("rollout_percentage") for condition in self.conditions),
             "filter_count": filter_count,
             "created_at": self.created_at,
+            "aggregating_by_groups": self.aggregation_group_type_index is not None,
```

Comment:
I don't see anything referencing `aggregating_by_groups`, should there be a test that depends on this?